x/xstrings/strings.go

```diff
@@ -0,0 +1,11 @@
+package xstrings
+
+// ContainsString returns true if slice s contains e element, false otherwise.
+func ContainsString(a []string, e string) bool {
```

Comment:
would be better rename to _ SliceContains()_

Reply:
ContainsString was chosen based on similarity to strings/bytes pkg:
Contains(s, substr string) 
ContainsAny(s, chars string) 
ContainsRune(s string, r rune)

Reply:
I see but in our case String word in the function name in xstrings package is not necessary because we define the package functions for the same context with package name. And for this case, it'd be good to indicate we're working on slices otherwise it looks like we're trying to find a substring in a string with current naming.